assets/css/activation.scss

```diff
@@ -1,10 +1,10 @@
 /*
 Plugin Name: Sensei
-Plugin URI: http://woothemes.com/
+Plugin URI: https://woocommerce.com/products/sensei/
```

Comment:
Did you happen to see [my comment](https://github.com/Automattic/sensei/issues/2198#issuecomment-411849134)? We can drop this header from these files altogether and just keep the description.